src/context/LocationsContext.js

```diff
@@ -26,8 +32,28 @@ const LocationsContextProvider = (props) => {
       .catch(console.log);
   }, []);
 
+  /*useEffect for when the recenter changes which happens in the map component, 
+  the api is call the api & the new locations are set */
```

Comment:
> the api is call the api

Could you clean this up a bit? I _think_ I know what you mean, but it’s a little confusing.

Reply:
yes! Sorry about the errors!